metrics/validate_drp.yaml

```diff
@@ -0,0 +1,223 @@
+# validate_drp metric definitions
+# Definitions from LPM-17 (the Science Requirements Document).
+# http://ls.st/lpm-17
+
+PA1:
+  description: >
+    The maximum rms of the unresolved source magnitude distribution around the
+    mean value.
+  unit: mmag
+  reference:
+    doc: LPM-17
+    url: http://ls.st/lpm-17
+    page: 21
+
+PF1_minimum_gri:
+  description: >
+    The maximum fraction of magnitudes deviating by more than PA2_minimum_gri
+    from mean (%).
+  unit: ''
```

Comment:
Hmmm... We should probably poke astropy about this: see [here](http://docs.astropy.org/en/stable/units/standard_units.html#the-dimensionless-unit), where they say "...though using u.dimensionless_unscaled in Python code is preferred for readability..."

That said, there's also [this Closed Issue](https://github.com/astropy/astropy/issues/4621).

It sounds like we should probably file an Issue to clarify their docs about dimensionless units.